compiler/qsc_frontend/src/incremental/tests.rs

```diff
@@ -11,40 +11,72 @@ use miette::Diagnostic;
 fn one_callable() {
     let store = PackageStore::new(compile::core());
     let mut compiler = Compiler::new(&store, vec![], TargetProfile::Full);
-    let fragments = compiler
+    let unit = compiler
         .compile_fragments(
             &mut CompileUnit::default(),
             "test_1",
             "namespace Foo { operation Main() : Unit {} }",
         )
         .expect("compilation should succeed");
 
-    check_fragment_kinds(
+    check_unit(
         &expect![[r#"
-        [
-            "callable",
-            "namespace",
-        ]
-    "#]],
-        &fragments,
+            ast:
+            Package 0:
+                Namespace 1 [0-44] (Ident 2 [10-13] "Foo"):
+                    Item 3 [16-42]:
+                        Callable 4 [16-42] (Operation):
+                            name: Ident 5 [26-30] "Main"
+                            input: Pat 6 [30-32]: Unit
+                            output: Type 7 [35-39]: Path: Path 8 [35-39] (Ident 9 [35-39] "Unit")
+                            body: Block: Block 10 [40-42]: <empty>
+            names:
+            node_id:2,node_id:5,node_id:8,
+            terms:
+            node_id:6,node_id:10,
+            hir:
+            Package:
+                Item 0 [0-44] (Public):
+                    Namespace (Ident 5 [10-13] "Foo"): Item 1
+                Item 1 [16-42] (Public):
+                    Parent: 0
+                    Callable 0 [16-42] (operation):
+                        name: Ident 1 [26-30] "Main"
+                        input: Pat 2 [30-32] [Type Unit]: Unit
+                        output: Unit
+                        functors: empty set
+                        body: SpecDecl 3 [16-42]: Impl:
+                            Block 4 [40-42]: <empty>
+                        adj: <none>
+                        ctl: <none>
+                        ctl-adj: <none>"#]],
+        &unit,
     );
 }
 
 #[test]
 fn one_statement() {
     let store = PackageStore::new(compile::core());
     let mut compiler = Compiler::new(&store, vec![], TargetProfile::Full);
-    let fragments = compiler
+    let unit = compiler
         .compile_fragments(&mut CompileUnit::default(), "test_1", "use q = Qubit();")
         .expect("compilation should succeed");
 
-    check_fragment_kinds(
+    check_unit(
         &expect![[r#"
-            [
-                "statement",
-            ]
-        "#]],
-        &fragments,
+            ast:
+            Package 0:
+                Stmt 1 [0-16]: Qubit (Fresh)
+                    Pat 2 [4-5]: Bind:
+                        Ident 3 [4-5] "q"
+                    QubitInit 4 [8-15] Single
+            names:
+            node_id:3,
+            terms:
+            node_id:1,node_id:2,node_id:3,node_id:4,
+            hir:
+            Package:"#]],
```

Comment:
Looks like this test output is missing it's HIR package content for some reason? The fact that it has `Package: ` means it is getting an HIR object, but it seems to be missing the top-level qubit allocation statement. Is the display for top-level stmts missing?

Reply:
Good catch, I thought I went over the new test output but I missed this. Investigating...